code/SecureFileController.php

```diff
@@ -139,12 +145,16 @@ public function sendFile($file) {
 			return file_get_contents($path);
 		}
 
-		$disposition = $this->config()->content_disposition;
-		if(!$disposition) $disposition = 'attachment';
+		if(!$disposition) {
+			$disposition = $this->config()->content_disposition;
+			if (!$disposition) $disposition = 'attachment';
```

Comment:
This breaks our coding style rules. A ternary operation would be more appropriate `$disposition = $this->config()->content_disposition ?: 'attachment';`

Reply:
I didn't change style of code that I didn't add or remove, but I'll update it.